nbviewer/providers/local/handlers.py

```diff
@@ -104,30 +141,22 @@ def get(self, path):
         path: str
             Local filesystem path
         """
-        abspath = os.path.abspath(os.path.join(
-            self.localfile_path,
-            path
-        ))
-
-        if not abspath.startswith(self.localfile_path):
-            app_log.warn("directory traversal attempt: '%s'" %
-                         self.localfile_path)
```

Comment:
Note: I've moved this up to the factorised function, but have improved the message so that it tells you the ```file_path``` that has actually been requested (not the invariant localfile root).